### PR TITLE
Fix: Initial usability pass

### PR DESCRIPTION
**Pull request by sapegin**
See #4 for details

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,4 +12,5 @@ Thumbs.db
 .idea
 .vscode
 *.sublime-project
-*.sublime-workspace
\ No newline at end of file
+*.sublime-workspace
```
Comment (bebraw) [inline .gitignore]: Shouldn't editor specific files be ignored on system level instead?

File: src/tasks/travis.js [hunk 2]
```diff
@@ -19,8 +22,17 @@ module.exports = () => {
         'node --version',
       ],
       before_script: [
-        'if [ "$WEBPACK_VERSION" ]; then yarn add webpack@^$WEBPACK_VERSION; fi',
-        'if [ "$BITHOUND_CHECK" ]; then npm install -g bithound; bithound check git@github.com:$TRAVIS_REPO_SLUG.git; fi',
+        `
+if [ "$WEBPACK_VERSION" ]; then
```
Comment (bebraw) [inline src/tasks/travis.js]: Maybe comment where WEBPACK_VERSION comes from.

File: .gitignore (hunk 1 above)
Comment (sapegin) [inline .gitignore]: I used to think that they should. But in this case you have to explain your opinion to all people who:

1. Commit their editor settings.
2. Commit changes to `.gitignore`.

I gave up 😞

File: src/tasks/travis.js (hunk 2 above)
Comment (sapegin) [inline src/tasks/travis.js]: In actual `.travis.yml` it’s just a few lines earlier.

File: .gitignore (hunk 1 above)
Comment (bebraw) [inline .gitignore]: Ok, fair enough. Perhaps that's better on personal projects then. 👍

Comment (d3viant0ne) [inline .gitignore]: Most people don't know to ignore that stuff at a system level. Most people don't even know they have a global git config. For the sake of the masses, probably best to ignore the editor files on a project level.

**Pull request merged**